app_server.py

```diff
@@ -246,8 +247,20 @@ def handle(self, exception):
 
 class HeartbeatController(object):
 
-    def heartbeat(self):
-        return make_response("", 200, {"Content-Type": "application/json"})
+    HEALTH_CHECK_TYPE = 'application/vnd.health+json'
+    VERSION_FILENAME = '.version'
+
+    def heartbeat(self, conf_class=None):
+        health_check_object = dict(status='pass')
+
+        Conf = conf_class or Configuration
+        app_version = Conf.app_version()
+        if app_version and app_version != Conf.NO_APP_VERSION_FOUND:
+            health_check_object['releaseID'] = app_version
+            health_check_object['version'] = app_version.split('-')[0]
```

Comment:
I think it would be useful to provide the whole thing here. The client can always decide to only show part of the version number.

Alternatively, I would change this code to `.split('-', 1)[0]`. That will do the same thing in the normal case, but will also work in the weird case where the full app version doesn't contain a dash.

Reply:
The Internet-Draft says that 'version' should represent the official release and 'releaseID' should represent any other changes. In the case of `git describe` the additional values would be the number of commits from that last official version and the commit sha at HEAD, which I think constitute patches / releaseID. (In production, releaseID and version will likely be the same. It's in the `development` containers or production hot-fixes that would show a difference.)

`split` returns an array with only one value even without the second parameter. (1 might be the default?)

Reply:
Well, you've got me there!